### PR TITLE
RFC 6265bis: SameSite -- clarify request method on redirects

### PR DESCRIPTION
**Pull request by chlily1**
This clarifies the question raised in issue #773 about SameSite cookies on redirects.

### CODE REVIEW

**Comment by chlily1** [comment]
@mikewest PTAL, thanks!

**Comment by reschke** [comment]
The request method is only updated for *some* of the redirects, and only if it is POST.

File: draft-ietf-httpbis-rfc6265bis.md [hunk 1]
```diff
@@ -1278,7 +1278,9 @@ drop-in mechanism that mitigates the risk of CSRF attacks, developers may set
 the `SameSite` attribute in a "Lax" enforcement mode that carves out an
 exception which sends same-site cookies along with cross-site requests if and
 only if they are top-level navigations which use a "safe" (in the {{RFC7231}}
-sense) HTTP method.
+sense) HTTP method. (Note that a request's method is updated on redirects, such
+that "safe"ness is determined based on the method of the most recent redirect
+hop.)
```
Comment (mikewest) [inline draft-ietf-httpbis-rfc6265bis.md:1283]: How about the following change to incorporate @reschke's suggestion:

```suggestion
sense) HTTP method. (Note that a request's method may be changed from POST
to GET for some redirects (see sections 6.4.2 and 6.4.3 of {{RFC7231}}); in these
cases, a request's "safe"ness is determined based on the method of the current
redirect hop.)
```

**Comment by mikewest** [comment]
The change generally looks fine. I'd suggest that adding tests would be helpful to ensure we guide implementations towards interoperable behavior.

**Pull request merged**